NorthstarDedicatedTest/NorthstarDedicatedTest.vcxproj.filters

```diff
@@ -121,6 +121,12 @@
     <Filter Include="Header Files\include\libcurl">
       <UniqueIdentifier>{ea1e17a6-40b7-4e1b-8edb-e9ae704ce604}</UniqueIdentifier>
     </Filter>
+    <Filter Include="Source Files\Shared\Math">
+      <UniqueIdentifier>{59b0f68f-daa7-4641-b6fa-8464b56da2bb}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="Header Files\Shared\Math">
+      <UniqueIdentifier>{44a83740-9d70-480d-9a7a-43b81f8eab9e}</UniqueIdentifier>
+    </Filter>
```

Comment:
Do we really want even more filters? Although, I guess we can just deal with them when we do the big refactor.

Reply:
For the extra files i added to this project, i don't think there was a better filter. I could place them elsewhere (in a existing filter) if desired, but i recommend a full project restructure regardless in the future.